barclayscrape.js

```diff
@@ -44,12 +44,13 @@ function login(casper, loginOpts) {
                 'membershipNumber': config.membership_number
             });
             this.click('button#forward');
-            this.waitForSelector('input#card-digits', function loginStageTwo() {
+            this.waitForSelector('input#passcode-radio', function loginStageTwo() {
```

Comment:
This change actually breaks my login flow (using pinsentry), which is still working. Changing that selector to look for either of them (i.e. `input#card-digits, input#passcode-radio`) seems to work though. If you can change that, I'll merge it.